src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background-color: $blue_accent;
+  border-radius: 5px;
+  color: $white;
+  transition: background 0.3s, color 0.3s;
```

Comment:
Save transition duration(0.3s) as a variable as it should be the same for all transitions

Reply:
Not fixed